Bottom: 51e7d22254f5ec84fd72591aa24b42715adefd1b
Top:    2a540220b1caae6d593fb8381d7c99b70d20da7d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-13 01:16:11 +0100

Refresh of hid-gtk-remove-auto-pan-featur

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index e06cfa1..2d2d4c7 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -1017,9 +1017,6 @@ draw_crosshair (GdkGC *xor_gc, gint x, gint y)
   prev = Crosshair.shape;
 }
 
-#define VCW 16
-#define VCD 8
-
 static void
 show_crosshair (gboolean paint_new_location)
 {
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 49dc66e..d82d800 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -737,15 +737,10 @@ draw_crosshair (gint x, gint y, gint z)
   prev = Crosshair.shape;
 }
 
-#define VCW 16 /* Crosshair pan-marker width in pixels */
-#define VCD 8  /* Crosshair pan-marker depth in pixels */
-
 void
 ghid_show_crosshair (gboolean paint_new_location)
 {
   gint x, y, z;
-  int vcw = VCW * gport->view.coord_per_px;
-  int vcd = VCD * gport->view.coord_per_px;
   static int done_once = 0;
   static GdkColor cross_color;
 
diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index fbda3ac..b213d54 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -52,7 +52,6 @@
 
 #define TOOLTIP_UPDATE_DELAY 200
 
-static gint x_pan_speed, y_pan_speed;
 void
 ghid_port_ranges_changed (void)
 {
@@ -607,8 +606,6 @@ gint
 ghid_port_window_leave_cb (GtkWidget * widget, 
                            GdkEventCrossing * ev, GHidPort * out)
 {
-  Coord x0, y0, x, y, dx, dy, w, h;
-  
   /* printf("leave mode: %d detail: %d\n", ev->mode, ev->detail); */
 
   /* Window leave events can also be triggered because of focus grabs. Some
